src/core/reducers/addons.js

```diff
@@ -48,6 +49,7 @@ export default function addon(state = initialState, action) {
         newState[key] = {
           ...thisAddon,
           installURL: thisAddon.current_version.files[0].url,
+          is_restart_required: thisAddon.current_version.files[0].is_restart_required,
```

Comment:
What if we called this `isRestartRequired` to make it a bit more obvious that it's a mapped property?